villanovachile/pizza-app
# Update: Integrate Redux Store with Pizza Size component

The PizzaSize component options and radio buttons have been updated and are now integrated into the Redux Store. 

Testing Instructions:
1. Select each pizza size option radio input
2. Confirm (using either Redux Dev Tool or React Dev Tools) that each selection is updating the corresponding selection to "true", and all unselected options to "false".

Note: The "small" pizza size is deleted by default. 


https://user-images.githubusercontent.com/10015451/229955986-8057c4d7-d9cc-4f28-86d9-bd77c2dc24dd.mp4
